### PR TITLE
Fix: multiple row gaps for table card view

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

**Comment by coreyar** [comment]
I don't see what this PR does by looking at the new story. The new story is identical to With Custom column width story

File: src/components/v2/Table/index.stories.tsx [hunk 1]
```diff
@@ -38,6 +38,15 @@ const useStyles = () => {
         display: initial;
       }
     `,
+
+    /* multiple rows styles */
+
+    cardContentGrid: `
```
Comment (coreyar) [inline src/components/v2/Table/index.stories.tsx:44]: This style should not be global to the table. It should be implemented by the consumer of the table component

Comment (baomastr) [reply src/components/v2/Table/index.stories.tsx:44]: it was in previous iteration, but was updated last week. 
so how do you see the implementation of this?

Comment (coreyar) [reply src/components/v2/Table/index.stories.tsx:44]: Table responsive styles are passed down to the table when implementing the table. This is done and you can see examples in the market tables

Comment (baomastr) [reply src/components/v2/Table/index.stories.tsx:44]: the discussion is currently in opened issue: https://app.clickup.com/t/2amjufw

**Comment by baomastr** [comment]
@coreyar @therealemjy the updates for Markets tables are ready for review

**Comment by therealemjy** [comment]
Looks good. I pushed a small fix for the padding of the table card container.

**Review by therealemjy — APPROVED** [review]

**Pull request merged**